### ISSUE TITLE
No local and federated timelines in app

### ISSUE THREAD

**Issue by LeonardoPaccianiMori** [mastodon/mastodon-android#359]
As the title suggests, the Android app only shows the "Home" timeline, while both the "Local" and "Federated" timelines are completely absent.

I'm not sure if this is intentional, but I think that at least the inclusion of the local timeline would make the mobile experience much better (I can see, though, how the federated timeline could be refreshing too fast for a nice user experience on a mobile app).

**Comment by York-Richter** [comment]
That was the first thing I missed when I opened the app. The first idea that I had too press the home icon to switch between the timelines, but...

**Comment by LeonardoPaccianiMori** [comment]
That's exactly what I thought the first time I opened the app, and I have to say I was pretty surprised that something like that was missing.

I'm also _**very**_ worried about the many users that I'm sure are getting into Mastodon _**exclusively**_ from the mobile app, especially these last weeks. Also for this reason I think it is absolutely important to add the local and federated timelines to the app.

**Comment by York-Richter** [comment]
ios user want that too
https://github.com/mastodon/mastodon-ios/issues/369

**Comment by York-Richter** [comment]
ios user want that too
https://github.com/mastodon/mastodon-ios/issues/369

**Comment by sk22** [comment]
see discussion in #8.
local timeline is actually there, but it's called "community" and it's placed inside the search tab.
you can, however, use my fork, which has this and more features that are missing in the official app: https://github.com/sk22/mastodos#readme

**Comment by LeonardoPaccianiMori** [comment]
I've been using the app for a few weeks now, and never noticed that. It's definitely not an intuitive placement for the local timeline. Thanks for the recommendation @sk22 , I'll check that out.
However, I think the official app should also support this since it doesn't seem like anything particularly complicated. If you look also at other third party apps like Tusky they all show the local and federated timelines in the lower bar besides the home timelime...

In short: the current placement of the local timeline is far from intuitive, and should be put in the lower bar besides the home timeline.

**Comment by York-Richter** [comment]
hi @sk22  thanks for the link, but I think for the most users is to install a unofficial APK not simple and not the preferred way. I deinstalled the Mastodon app and installed the PWA, but hope for the app users that the timelines come.

**Comment by sk22** [comment]
yes, definitely, i hope so too!

**Comment by nourkagha** [comment]
I strongly agree that local and federated tabs should be added at the bottom.

The current design is just confusing new users, especially when it's not a reflection of the web UI at all.

The decision to not include those timelines was to simplify things and not overwhelm new users, but everyone understands local and federated timelines.

It's awkward when you explain Mastodon and the different timelines to people and they join and can't find them, leaving them even more confused.

**Comment by LeonardoPaccianiMori** [comment]
I 100% agree with @nourkagha

**Comment by XaurDesu** [comment]
We should definitely add some sort of easy way to see/distinguish local and federated posts, people already familiar with Mastodon might use Tusky or some other external client, but people who just know Mastodon because of the whole twitter situation will probably just search 'Mastodon' in the play store and go from there. Given the growth of the platform, i'd imagine it's the sensible thing to do.

**Comment by YukiSnowmew** [comment]
@asklow I don't know about you, but those within my friend groups who have migrated to Mastodon have stayed. In fact, there's yet another massive migration wave from Twitter just today. We absolutely do have to worry about the experience of Twitter migrators.

Federated and local timelines are not niche. Every public post goes to the local timeline and each instance has rules about post visibility which limits what is allowed on local. It is absolutely critical that new users know about the local timeline so they aren't blindsided by moderators doing their jobs.

Here is a post of mine telling people to use Tusky et al.
https://meow.social/@YukiSnowmew/109370878308677222

Note the responses of those who used the official app before seeing my post. The response to switching away from the official app is *positive*. That should be concerning, since the official app is the first way new users will experience Mastodon. When an alternative is more intuitive, there's a problem.

**Comment by parkerhindman** [comment]
I’d also like to chime in and say that I think it’s silly to not at least have the option to load the federated timeline, the app isn’t reflective of the webui and makes the official app nearly unusable for smaller instances who don’t want random posts on their main timeline but still want an easy path to discoverability. Hashtags do not serve this purpose imo.

**Comment by nemobis** [comment]
See also https://github.com/mastodon/mastodon-android/issues/8#issuecomment-1126651906

**Comment by bburdette** [comment]
Having the app differ significantly in interface from the website is confusing.  The problem appears to be the Live Feeds feature might get the app banned.  Fair enough.  But instead of just leaving that part out, the Explore feed is also renamed as Search Mastodon.  Makes it confusing as to what I'm seeing in the app, as neither Explore nor Live Feeds is mentioned in the UI.  

Besides that, how about have an FDroid edition that has full features of the website, ie Live Feeds?

**Comment by kotroy** [comment]
figured I could access mastodon through my phone's browser (Chrome in this case) > three dots > "install $instance app". and boom, the interface is much better and easy to access. uninstalled the play store app right away
